packages/core/src/sdk/product/useProductGalleryQuery.ts

```diff
@@ -66,9 +72,31 @@ export const useProductGalleryQuery = ({
     selectedFacets,
   })
 
+  const findFacetValue = (
+    facets: Facet[],
+    searchParam: string
+  ): string | null => {
+    const facet = facets.find(({ key }) => key === searchParam)
+    return facet?.value ?? null
+  }
+
   return useQuery<Query, Variables>(query, localizedVariables, {
     onSuccess: (data) => {
       if (data && term) {
+        const fuzzyFacetValue = findFacetValue(selectedFacets, 'fuzzy')
```

Comment:
could the `term` influence this logic?  maybe we could move the `if(term){...}` just for the rest of the logic internally (sendAnalyticsEvent).

Reply:
it does, because we just want to send this parameters when we are using the search (looking for a term). We shouldn't send it when going directly the category PLP.